tests/unit/Mocks/TransactionMock.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
Mesmo cenário de `TransactionsListMock`: o que acha de trabalharmos com arquivos JSON ao invés de usar classes diretamente para retornar os valores de mocks?

Reply:
E como pegariamos o conteúdo? Se for com `file_get_contents` achei que ficaria um syntax um pouco ruim, aí fiz assim pq achei que ficaria mais facil de entender.